l_leg_controller:
  type: "robot_mechanism_controllers/JointTrajectoryActionController"
  joints:
    - l_leg_uhz
    - l_leg_mhx
    - l_leg_lhy
    - l_leg_kny
    - l_leg_uay
    - l_leg_lax
  gains:
    l_leg_uhz: {p:  5000.0, d: 10.0, i: 0.0, i_clamp: 0.0}
    l_leg_mhx: {p: 50000.0, d: 10.0, i: 0.0, i_clamp: 0.0}
    l_leg_lhy: {p: 20000.0, d: 10.0, i: 0.0, i_clamp: 0.0}
    l_leg_kny: {p:  1000.0, d: 10.0, i: 0.0, i_clamp: 0.0}
    l_leg_uay: {p:  1000.0, d: 10.0, i: 0.0, i_clamp: 0.0}
    l_leg_lax: {p:   500.0, d: 10.0, i: 0.0, i_clamp: 0.0}
  joint_trajectory_action_node:
    joints:
      - l_leg_uhz
      - l_leg_mhx
      - l_leg_lhy
      - l_leg_kny
      - l_leg_uay
      - l_leg_lax
    constraints:
      goal_time: 0.02
      l_leg_uhz:
        goal: 0.02
      l_leg_mhx:
        goal: 0.02
      l_leg_lhy:
        goal: 0.02
      l_leg_kny:
        goal: 0.02
      l_leg_uay:
        goal: 0.02
      l_leg_lax:
        goal: 0.02

r_leg_controller:
  type: "robot_mechanism_controllers/JointTrajectoryActionController"
  joints:
    - r_leg_uhz
    - r_leg_mhx
    - r_leg_lhy
    - r_leg_kny
    - r_leg_uay
    - r_leg_lax
  gains:
    r_leg_uhz: {p:  5000.0, d: 10.0, i: 0.0, i_clamp: 0.0}
    r_leg_mhx: {p: 50000.0, d: 10.0, i: 0.0, i_clamp: 0.0}
    r_leg_lhy: {p: 20000.0, d: 10.0, i: 0.0, i_clamp: 0.0}
    r_leg_kny: {p:  1000.0, d: 10.0, i: 0.0, i_clamp: 0.0}
    r_leg_uay: {p:  1000.0, d: 10.0, i: 0.0, i_clamp: 0.0}
    r_leg_lax: {p:   500.0, d: 10.0, i: 0.0, i_clamp: 0.0}
  joint_trajectory_action_node:
    joints:
      - r_leg_uhz
      - r_leg_mhx
      - r_leg_lhy
      - r_leg_kny
      - r_leg_uay
      - r_leg_lax
    constraints:
      goal_time: 0.02
      r_leg_uhz:
        goal: 0.02
      r_leg_mhx:
        goal: 0.02
      r_leg_lhy:
        goal: 0.02
      r_leg_kny:
        goal: 0.02
      r_leg_uay:
        goal: 0.02
      r_leg_lax:
        goal: 0.02
